sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -323,6 +323,40 @@ To see how to use the Document Translation client library with Azure Storage Blo
 for your containers, and download the finished translated documents, see this [sample][sample_translation_with_azure_blob].
 Note that you will need to install the [azure-storage-blob][azure_storage_blob] library to run this sample.
 
+## Advanced Topics
+
+The following section provides some insights for some of the advanced translation features such as glossaries and custom translation models.
+
+### **Glossaries**
+Glossaries are domain-specific dictionaries. For example, if you want to translate some medical-related documents, you may need support for the many words, terminology, and idioms in the medical field which you can't find in the standard translation dictionary or you simply need specific translation. This is why Document Translator provides support for glossaries. 
+
+#### **How To Create Glossary File**
+
+Document Translator supports glossaries in the following formats:
+
+|**File Type**|**Extension**|**Description**|
```

Comment:
it looks like they also support csv files for glossaries: https://docs.microsoft.com/en-us/azure/cognitive-services/translator/document-translation/overview#supported-glossary-formats